packages/hello-world/.gitignore

```diff
@@ -0,0 +1,3 @@
+dist
+node_modules
+build.tar
```

Comment:
for some reason the generated project doesn't have this `.gitignore` file.

Reply:
Ah, nice catch – will fix on Monday.

Reply:
Oh, this is true for all projects, not just hello-world. I think NPM must exclude files named `.gitignore` from a package upload (the full project source code is bundled with the generator, and always has been).